pkgs/by-name/ut/util-linux/package.nix

```
@@ -190,13 +191,15 @@ stdenv.mkDerivation (finalAttrs: {
     prefix=$swap _moveSbin
     ln -svf "$swap/bin/"* $bin/bin/
   ''
-  # moveToOutput "lib/liblastlog2*" "$lastlog"
   + lib.optionalString withLastlog ''
     ${lib.optionalString (!stdenv.hostPlatform.isStatic) ''moveToOutput "lib/security" "$lastlog"''}
     moveToOutput "lib/tmpfiles.d/lastlog2-tmpfiles.conf" "$lastlog"
 
     moveToOutput "bin/lastlog2" "$lastlog"
     ln -svf "$lastlog/bin/"* $bin/bin/
+
+    substituteInPlace $lastlog/lib/systemd/system/lastlog2-import.service \
+      --replace-fail "/usr/bin/mv" "${lib.getExe' coreutils "mv"}"
   ''
   + lib.optionalString (withLastlog && systemdSupport) ''
     moveToOutput "lib/systemd/system/lastlog2-import.service" "$lastlog"
```

```suggestion
  ''
  + lib.optionalString (withLastlog && systemdSupport) ''
    moveToOutput "lib/systemd/system/lastlog2-import.service" "$lastlog"
    substituteInPlace $lastlog/lib/systemd/system/lastlog2-import.service \
      --replace-fail "/usr/bin/mv" "${lib.getExe' coreutils "mv"}"
```
otherwise this breaks